Sources/UAPusher/Builder/UABuilder.swift

```diff
@@ -0,0 +1,245 @@
+import Vapor
+
+/// Builder class for the request payload
+public final class UABuilder {
+    
+    // MARK: Class fields
+    
+    var audiencePayload: Audience?
```

Comment:
Would it be simpler if this basically just had a variable with `[BuildingBlock]` instead?